### DISCUSSION TITLE
correct the spacing between the tabs in the navigation

### DISCUSSION THREAD

**Discussion by dua-arpit** [alshedivat/al-folio#2595] [Q&A]
How does this space appear between the Publications and Teaching tabs? 

![image](https://github.com/user-attachments/assets/9173f10e-5d97-4588-88fb-35b224bbdafd)

Is it due to the navigation order?

**Comment by george-gca** [comment]
How did you remove the blog? Maybe it has something to do with it.

**Reply by dua-arpit** [reply]
I changed _nav_ to _false_ in blog.md

**Reply by george-gca** [reply]
Humm I guess we are missing some cleaning up in this part of the code. Would you mind opening an issue for this, or even sending a PR to fix this? This issue is probably fixed by changing some of the `{% %}` by `{%- -%}` in [_includes/header.liquid](https://github.com/alshedivat/al-folio/blob/2db33ea99f04f8769207d85ad24b56160496f7ba/_includes/header.liquid#L62) like referred to on #1502.

**Comment by george-gca** [comment]
Closed as not a bug in #2596
